### ISSUE TITLE
Post Task Failing - Colima 0.6

### ISSUE THREAD

**Issue by madhavajay** [crazy-max/ghaction-setup-docker#45]
### Behaviour

The post task started failing:
https://github.com/OpenMined/PySyft/actions/runs/6845889094/job/18611767230?pr=8246

```
Post job cleanup.
Docker daemon logs
  /usr/local/bin/colima exec -- cat /var/log/docker.log
  cat: /var/log/docker.log: No such file or directory
  time="[2](https://github.com/OpenMined/PySyft/actions/runs/6845889094/job/18611767230?pr=8246#step:21:2)02[3](https://github.com/OpenMined/PySyft/actions/runs/6845889094/job/18611767230?pr=8246#step:21:3)-11-13T05:21:38Z" level=fatal msg="exit status 1"

/Users/runner/work/_actions/crazy-max/ghaction-setup-docker/v2.1.0/webpack:/docker-setup-docker/node_modules/@actions/exec/lib/toolrunner.js:592
                error = new Error(`The process '${this.toolPath}' failed with exit code ${this.processExitCode}`);
^
Error: The process '/usr/local/bin/colima' failed with exit code 1
    at ExecState.processExited [as _setResult] (/Users/runner/work/_actions/crazy-max/ghaction-setup-docker/v2.1.0/webpack:/docker-setup-docker/node_modules/@actions/exec/lib/toolrunner.js:592:1)
    at ExecState.exitCode [as CheckComplete] (/Users/runner/work/_actions/crazy-max/ghaction-setup-docker/v2.1.0/webpack:/docker-setup-docker/node_modules/@actions/exec/lib/toolrunner.js:577:1)
    at ChildProcess.<anonymous> (/Users/runner/work/_actions/crazy-max/ghaction-setup-docker/v2.1.0/webpack:/docker-setup-docker/node_modules/@actions/exec/lib/toolrunner.js:[4](https://github.com/OpenMined/PySyft/actions/runs/6845889094/job/18611767230?pr=8246#step:21:4)71:1)
    at ChildProcess.emit (node:events:[5](https://github.com/OpenMined/PySyft/actions/runs/6845889094/job/18611767230?pr=8246#step:21:5)14:28)
    at maybeClose (node:internal/child_process:1105:1[6](https://github.com/OpenMined/PySyft/actions/runs/6845889094/job/18611767230?pr=8246#step:21:7))
    at Process.ChildProcess._handle.onexit (node:internal/child_process:305:5)
```

#### Steps to reproduce this issue

1. I have tried both `uses: crazy-max/ghaction-setup-docker@v2.1.0` and `uses: crazy-max/ghaction-setup-docker@v2.0.0`
2. I think this is related to `colima==0.6.0`

#### Expected behaviour
The tasks pass so the final post hook shouldn't fail the job.

> Tell us what should happen

#### Actual behaviour

> Tell us what happens instead

### Configuration

* Repository URL (if public): 
* https://github.com/OpenMined/PySyft/actions/runs/6845889094/job/18611767230?pr=8246
* Build URL (if public): 

```yml
- name: Docker on MacOS
    if: steps.changes.outputs.syft == 'true' && matrix.os == 'macos-latest'
    uses: crazy-max/ghaction-setup-docker@v2.1.0

  - name: Run unit tests
    if: steps.changes.outputs.syft == 'true'
    run: |
      tox -e syft.test.unit
```

### Logs

> Download the [log file of your build](https://docs.github.com/en/actions/managing-workflow-runs/using-workflow-run-logs#downloading-logs)
> and [attach it](https://docs.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests) to this issue.

**Comment by madhavajay** [comment]
I can verify that by reverting from `macos-13` to `macos-latest` I can use `colima==0.5.6` and the issue goes away.

I don't know why that `/var/logs/docker.log` file is no longer in the colima vm or where ever its getting cat'ed from but that might be the easiest fix.

**Comment by crazy-max** [comment]
It might be linked to the image defaulting to Ubuntu since 0.6.0: https://github.com/OpenMined/PySyft/actions/runs/6845889094/job/18611767230?pr=8246#step:11:543

```
time="2023-11-13T05:09:52Z" level=info msg="Attempting to download the image" arch=x86_64 digest="sha512:ff16319abfd9b5e81395b0d0e7058a61b4bbd057fbab40fe654fb38cac1ed53b53a6a6d4c8ad10802f800daf5e069597e89ec2b2d360dbbc1dc44d0c2b1372fa" location="https://github.com/abiosoft/colima-ubuntu/releases/download/v0.6.0/ubuntu-23.10-minimal-cloudimg-amd64.img"
  Downloading the image (ubuntu-23.10-minimal-cloudimg-amd64.img)
```

With 0.5.6 it's Alpine: https://github.com/crazy-max/ghaction-setup-docker/actions/runs/6833543039/job/18585413192#step:3:221

```
  time="2023-11-11T10:02:57Z" level=info msg="Attempting to download the image" arch=x86_64 digest="sha512:48bf6c7468fc8acc05d14b3b138958cf4417fa26e478d864c8458a0c7aa8e9742c19058f792debc7585614e0a4ba6ad9608c2e6ff695a2d7ae8daafb8ad64db2" location="https://github.com/abiosoft/alpine-lima/releases/download/colima-v0.5.6/alpine-lima-clm-3.18.3-x86_64.iso"
  Downloading the image (alpine-lima-clm-3.18.3-x86_64.iso)
```

I will take a look, thanks for your feedback.

**Comment by crazy-max** [comment]
https://github.com/abiosoft/colima/blob/main/docs/FAQ.md#version-v060-and-newer

> ### Version v0.6.0 and newer
> 
> Colima uses Ubuntu as the underlying image. Other distros are not supported.

Looks like other distros are not supported anymore :disappointed_relieved:

**Comment by crazy-max** [comment]
https://github.com/docker/actions-toolkit/pull/201 should fix this issue.

**Comment by crazy-max** [comment]
Fixed by #46
